### ISSUE TITLE
ShadowFix is hiding the Minimize/Maximize/Close buttons on Windows 8.1

### ISSUE THREAD

**Issue by LOuroboros** [overdodactyl/userContent-tweaks#302]
**Browser**: Mozilla Firefox v72.0.2 (32 Bits)
**OS**: Windows 8.1 x64
**How you are using ShadowFox**: I grabbed all the code from the [userchrome.css](https://github.com/overdodactyl/ShadowFox/blob/master/userChrome.css) and [usercontent.css](https://github.com/overdodactyl/ShadowFox/blob/master/userContent.css) files, dropped them in profile/chrome/shadowfox and then added an @include to them in my personal userchrome.css and usercontent.css files.

**Screenshots**:

Without Shadowfox:
![firefox-2020-01-30_13-07-32](https://user-images.githubusercontent.com/4485172/73467058-875b0780-4361-11ea-9dc2-b24a72bc9d95.png)

With ShadowFox:
![firefox-2020-01-30_13-07-14](https://user-images.githubusercontent.com/4485172/73467079-8cb85200-4361-11ea-83c7-2be1ab248520.png)

**Comment by reiny2000** [comment]
hey! i'm experiencing the same thing on windows 7 since yesterday 10 may 2020. have you found a fix for this? using firefox nightly 78.0a1

**Comment by LOuroboros** [comment]
> 
> 
> hey! i'm experiencing the same thing on windows 7 since yesterday 10 may 2020. have you found a fix for this? using firefox nightly 78.0a1

Nope, I haven't. I just try to ignore it, since I don't really need to use them.

**Comment by brsk93** [comment]
This happens to me if I try to set gfx.webrender.enabled to true on Windows 7.




> 
> 
> hey! i'm experiencing the same thing on windows 7 since yesterday 10 may 2020. have you found a fix for this? using firefox nightly 78.0a1

Did you update nightly from 77? I'm using stable but if that's the case I'll need to remember not to update firefox after 77. Maybe disabling webrender will work for you.

**Comment by Albirew** [comment]
same behavior here on win7, firefox 82 dev edition, latest shadowfox release and gfx.webrender.enabled=false
EDIT: fixed by deleting this line in userchrome.css: `background-color: var(--tab-background-color)!important;`
